### DISCUSSION TITLE
Where can I read reference about all the built-in functions?

### DISCUSSION THREAD

**Discussion by Daesina** [dialogic-godot/dialogic#1921] [Q&A] [answered]
thank you

**Comment by CakeVR** [comment]
Hello, here is our online documentation: https://dialogic-docs.coppolaemilio.com/README/
It's still work in progress. The tabs/pages inside Dialogic have information too.

**Reply by Daesina** [reply]
hello. Thank you. Is there something for dialogic 1? And what are tabs/pages? Are they the scripts of dialogic that can be accessed through godot?

**Reply by Jowan-Spooner** [reply, marked as answer]
Dialogic 1 documentation is inside the plugin or in the repository [here](https://github.com/coppolaemilio/dialogic/blob/dialogic-1/addons/dialogic/Documentation/Content/Welcome.md).

With tabs/pages CakeVR was referring to the different editors and settings pages in dialogic2.
